docs/examples.rst

```diff
@@ -130,6 +130,29 @@ Or you can use a dict::
     }
     new_issue = jira.create_issue(fields=issue_dict)
 
+You can even bulk create multiple issues::
+
+    issue_list = [
+    {
+        'project': {'id': 123},
+        'summary': 'First issue of many',
+        'description': 'Look into this one',
+        'issuetype': {'name': 'Bug'},
+    },
+    {
+        'project': {'key': 'FOO'},
+        'summary': 'Second issue',
+        'description': 'Another one',
+        'issuetype': {'name': 'Bug'},
+    },
+    {
+        'project': {'name': 'Bar'},
+        'summary': 'Last issue',
+        'description': 'Final issue of batch.',
+        'issuetype': {'name': 'Bug'},
+    }]
+    issues = jira.create_issues(field_list=issue_dict)
```

Comment:
Good catch on the documentation error!

The return value has been redesigned. Please view the follow up commit to see the new return value that addresses this valid concern.